stdlib/public/runtime/SwiftValue.mm

```diff
@@ -430,10 +430,10 @@ - (BOOL)isEqual:(id)other {
     }
   }
 
-  if (runtime::bincompat::useLegacySwiftObjCHashing()) {
-    // Legacy behavior only proxies isEqual: for Hashable, not Equatable
-    return NO;
-  }
+//  if (runtime::bincompat::useLegacySwiftObjCHashing()) {
+//    // Legacy behavior only proxies isEqual: for Hashable, not Equatable
+//    return NO;
+//  }
```

Comment:
Can you remove the code outright rather than leave it commented? It's easy to revert a commit later if needed.

Reply:
Done.